tests/test_api.py

```diff
@@ -318,6 +318,14 @@ def test_upload_thumbnail_for_task(self):
             self.version['id'], {'image': None})
         expected_clear_thumbnail = {'id': self.version['id'], 'image': None, 'type': 'Version'}
         self.assertEqual(expected_clear_thumbnail, response_clear_thumbnail)
+        
+    def test_upload_thumbnail_with_upload_function(self):
+        """Upload thumbnail via upload function test"""
+        size = os.stat(path).st_size
```

Comment:
So we need the other two lines back in here:

```
this_dir, _ = os.path.split(__file__)
path = os.path.abspath(os.path.expanduser(
    os.path.join(this_dir,"sg_logo.jpg")))
```

as they're used to upload the thumbnail image.

the size variable doesn't actually get used further down.